core/models.py

```diff
@@ -1 +1,11 @@
+from django.db import models
 # Create your models here.
+
+
+class NextMeeting(models.Model):
+    title = models.CharField(max_length=50)
```

Comment:
Okey, I didn't see this when I was approving. I think you should increase max-length to at least 150. 50 for title is certainly too litt;e